### PR TITLE
invite member

### PR DESCRIPTION
**Pull request by nitinshr135**
Adds
- Invited Members tab in Members 
- Success or failure popup for invite mail
- error message in inviteCallback

### CODE REVIEW

File: src/components/Dashboard/routes/InviteMembers/InviteMembers.tsx [hunk 1]
```diff
@@ -26,12 +64,28 @@ const InviteMembers = () => {
...
       (res) =>
         res.subscribe((data) => {
+          setInviteData(data.success);
           setInviteMembersLoading(false);
-          history.push("/dashboard/members");
         }),
     );
   };
 
+  // const emailChecker = () => {
```
Comment (rekpero) [inline src/components/Dashboard/routes/InviteMembers/InviteMembers.tsx:73]: Remove this if not used

File: src/components/Dashboard/routes/InviteMembers/InviteMembers.tsx [hunk 2]
```diff
@@ -58,18 +112,91 @@ const InviteMembers = () => {
...
+                      Send
+                    </button>
+                  </div>
+                }
+                position="center center"
+                className="invite-popup"
+                modal
+                nested
+              >
+                {!inviteMemberLoading ? (
```
Comment (rekpero) [inline src/components/Dashboard/routes/InviteMembers/InviteMembers.tsx:150]: Put this in a diff component and pass it here

File: src/components/Dashboard/routes/Members/Members.tsx [hunk 3]
```diff
@@ -1,19 +1,29 @@
...
 const Members = () => {
   const history = useHistory();
-  const { userLoading, selectedOrg, orgLoading } =
-    useContext<IStateModel>(StateContext);
+  const { selectedOrg, orgLoading } = useContext<IStateModel>(StateContext);
   const [memberLoading, setMemberLoading] = useState<boolean>(false);
   const [members, setMembers] = useState<IMemberModel[]>([]);
+  const [invitedMembers, setInvitedMembers] = useState<IInviteMemberModel[]>([]);
+  const { fetchUser } = useContext<IActionModel>(ActionContext);
+  // const [memberDeleted, setMemberDeleted] = useState<boolean>();
```
Comment (rekpero) [inline src/components/Dashboard/routes/Members/Members.tsx:26]: remove this

File: src/components/Dashboard/routes/Members/Members.tsx [hunk 4]
```diff
@@ -38,6 +58,27 @@ const Members = () => {
     }
   }, [selectedOrg, orgLoading]);
 
+  const deleteInvitedUser = (userId: string) => {
+    ApiService.deleteInvite(userId).subscribe((result) => {
+      fetchUser();
+    });
+  };
+
+  // useEffect(() => {
```
Comment (rekpero) [inline src/components/Dashboard/routes/Members/Members.tsx:67]: Remove this

File: src/services/ApiService.ts [hunk 5]
```diff
@@ -141,6 +141,35 @@ export const updateInvite = (inviteReply: any): Observable<any> => {
   });
 };
 
+// export const getInviteList = (organizationId: string): Observable<any> => {
```
Comment (rekpero) [inline src/services/ApiService.ts:144]: remove this

**Review by rekpero — CHANGES_REQUESTED** [review]
LGTM!! @nitinshr135 please clean up the code before raising a PR

File: src/components/InvitePopup/InvitePopup.tsx [hunk 6]
```diff
@@ -0,0 +1,94 @@
+import React, { useContext } from "react";
```
Comment (rekpero) [inline src/components/InvitePopup/InvitePopup.tsx:1]: Put this component in the InviteMember component as it is connected to only that.

**Review by rekpero — CHANGES_REQUESTED** [review]
@nitinshr135 fix this asap. everything else looks good to go

File: src/components/InvitePopup/InvitePopup.tsx (hunk 6 above)
Comment (nitinshr135) [reply src/components/InvitePopup/InvitePopup.tsx]: yes, changes made

**Review by rekpero — APPROVED** [review]
LGTM

**Pull request merged**